exceptions/v1/notices.json

```diff
@@ -3,6 +3,17 @@
   "id": "exceptions/v1/notices.json",
   "type": "object",
   "properties": {
+    "correlation_context": {
+      "id": "#/properties/correlation_context",
+      "type": "object",
+      "properties": {
+        "request_id": {
```

Comment:
Would it make sense to add a "correlation id" property as well here? Or do we want to do some more thinking about it before adding such property?

The property name could be `id` (as it's already under `correlation_context`) or it could be the full name `correlation_id`.

Reply:
We have talked briefly about a `correlation_id` but I think we should focus on a `request_id` for now, as that is the term most clients use (we are using it in 2 clients now). It could also be confusing to have both concepts.

Tracing functionality might be the way we introduce a more generic correlation abstraction, which we would also store in this bucket.